Buildings/Controls/OBC/ASHRAE/G36/Atomic/SupplyAirTempSet_MultiZone.mo

```diff
@@ -0,0 +1,330 @@
+within Buildings.Controls.OBC.ASHRAE.G36.Atomic;
+block SupplyAirTempSet_MultiZone
```

Comment:
As this is only for VAV systems, the block should have `VAV` in its name, or somewhere in its package name.

Reply:
@JayHuLBL could this block have a name equivalent to the single zone one (without the fan speed)?